components/Breadcrumb/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@gemeente-denhaag/breadcrumb",
+  "description": "A Breadcrumb component",
+  "author": "Municipality of The Hague",
+  "license": "EUPL-1.2",
+  "version": "0.2.3-alpha.63",
```

```suggestion
  "version": "0.1.0",
```

This is a new component, so it's versioning should be new too :)

---

Copied form the Buttons or Link, forgot to change this (also didn't knew we applied a version before it's fully completed?).

---

The CI will bump it to something like `0.1.1-alpha.0` :D